client/landing/jetpack-cloud/components/log-item/style.scss

```diff
@@ -0,0 +1,72 @@
+.log-item {
+	border-radius: 3px;
+
+	&__header-wrapper {
+		display: flex;
+		flex-wrap: wrap;
+		align-items: center;
+	}
+
+	&__tag {
+		display: block;
+		margin-right: 10px;
+		padding: 2px 6px;
+		border-radius: 3px;
+		background-color: var( --color-primary-20 );
+		font-size: 10px;
+		color: var( --color-primary-0 );
+		text-transform: uppercase;
+		text-align: center;
+		min-width: 48px;
+		font-weight: 600;
+		height: 16px;
+	}
+
+	&__header-wrapper .card-heading {
+		font-weight: 600;
+		margin: 0;
+		color: var( --color-primary-20 );
+	}
+
+	&__subheader {
+		flex-basis: 100%;
+		margin: 0;
+	}
+
+	&.is-expanded .foldable-card__content {
+		border-top-width: 0;
+	}
+
+	&.is-error {
+		.log-item {
+			&__tag {
```

Comment:
I got a bit confused when first reading this nested selector. While I'm aware they will eventually get resolved by a sass engine to a simpler form, they are quite hard to read and understand here.

How about simplifying the selectors to something more verbose and obvious like:

```scss
&.is-error .log-item__tag {
  background-color: var( --color-error-20 );
}

&.is-error .log-item__header-wrapper .card-heading {
  color: var( --color-error-20 );
}
```

Reply:
+1 on this!

Reply:
I simplified in 3e59ab1a9826c0d71e25edb19f627d9a918f8fbb. Let me know if I can do anything further.